server/src/game/Game.hpp

```
@@ -40,6 +41,10 @@ namespace game {
 
       std::shared_ptr<Projectile> getProjectile(std::uint32_t projectile_id);
 
+      std::shared_ptr<ecs::ServerInputSystem> getServerInputSystem() {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)